src/main/java/com/yworks/yguard/obf/classfile/ClassFile.java

```diff
@@ -1751,4 +1751,12 @@ public String findModuleName() {
     }
     return "";
   }
+
+  public int getMajorVersion() {
```

Comment:
What's this code doing here? What are all the whitespace changes for?

Reply:
That's particle of my try for version check position. I will remove them.

Reply:
What are all the whitespace changes for?
=>
At that time, I was feeling a little strange because some files are not formatted. (brace, space.. etc)
So I just write my code and push it.
Can I reformat changed class file?
I'm using Intellij and it's look like recognize automatically .editorconfig file in yGuard project.

Reply:
You should not be reformatting unrelated source code. Even if the old code does not comply with the settings in .editorconfig that would mean you are changing the "blame" and "history" of the repository. 

We can do a reformat code in a separate commit, if we feel like this helps, but this should not be done as part of a minor unrelated code feature.